apps/aklite-offline/cmds.h

```diff
@@ -18,21 +18,28 @@ class Cmd {
  public:
   using Ptr = std::shared_ptr<Cmd>;
   const std::string& name() const { return _name; }
+  const std::string& description() const { return _description; }
   const po::options_description& options() const { return _options; }
 
   virtual int operator()(const po::variables_map& vm) const = 0;
 
  protected:
-  Cmd(std::string name, const po::options_description& options) : _name{std::move(name)}, _options{options} {}
+  Cmd(std::string name, std::string description, const po::options_description& options)
+      : _name{std::move(name)}, _description{std::move(description)}, _options{options} {}
 
  private:
   const std::string _name;
+  const std::string _description;
   const po::options_description& _options;
 };
 
 class CheckCmd : public Cmd {
  public:
-  CheckCmd() : Cmd("check", _options) {
+  CheckCmd()
+      : Cmd("check",
+            "Update the device TUF metadata reading and validating the metadata of the offline bundle. The list of "
```

Comment:
Maybe `Update the device TUF metadata by fetching and validating the offline bundle's metadata...`?

Reply:
Updated to the suggested text